images/gcp/vm_images.yaml

```diff
@@ -1,5 +1,5 @@
-gcp_foundation_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-gcp_extension_e1_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-gcp_extension_e2_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-gcp_extension_e4_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/aoc-aap-24-gcp-20230615174447
-image_internal_install_name: quay.io/ansible-on-clouds/ansible-on-clouds-internal-install:2.3-latest
\ No newline at end of file
+gcp_foundation_image_name: https://www.googleapis.com/compute/v1/projects/agcp-001-dev/global/images/cahl-aoc-gcp-2-4-200230828-00
```

Comment:
Id recommend not using personal images here, and instead use ones built via automation if possible.
Feel free to accept/reject this proposal.

Reply:
I will put official one there once we get one.  We were stuck on building images so I put a personal one in here.   Since I just merged my GCP PR and AWS PR, I should be able to get images built officially and can update.

Reply:
Do you mean that you plan on updating this soon within this PR? Or in a later PR?

Reply:
If later, feel free to resolve.